addons/website_livechat/models/website_visitor.py

```diff
@@ -73,6 +73,13 @@ def action_send_chat_request(self):
         if mail_channel_vals_list:
             mail_channels = self.env['mail.channel'].create(mail_channel_vals_list)
             # Open empty chatter to allow the operator to start chatting with the visitor.
-            mail_channels_info = mail_channels.channel_info('channel_minimize')
+            values = {
+                'fold_state': 'open',
+                'is_minimized': True,
+            }
+            domain = [('partner_id', '=', self.env.user.partner_id.id), ('channel_id.uuid', '=', mail_channels.uuid)]
```

Comment:
This will crash if there are multiple channels. Can't access fields in that case, need to use mapped and adapt query accordingly (using 'in', not having 'limit 1', etc)